src/razor/src/simplify/serializableDelegatedSimplifyMethodParams.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SerializableTextDocumentIdentifierAndVersion } from './serializableTextDocumentIdentifierAndVersion';
+import { TextEdit } from 'vscode-html-languageservice';
+
+export interface SerializableDelegatedSimplifyMethodParams {
+    identifier: SerializableTextDocumentIdentifierAndVersion;
```

Comment:
I forgot we changed to use `TextDocumentAndIdentifier`. @ryzngard do we need to update other types in the extension to this, or is the serialization compatible with whatever they currently have?